mlflow/sklearn/utils.py

```diff
@@ -310,7 +311,10 @@ def plot_confusion_matrix(*args, **kwargs):
                 "axes.labelsize": 10.0,
             }
         ):
-            return sklearn.metrics.plot_confusion_matrix(*args, **kwargs)
+            if Version(sklearn.__version__) >= Version("1.0"):
```

Comment:
It looks like we need to adjust the version here:

https://github.com/mlflow/mlflow/runs/6032083575?check_suite_focus=true#step:12:175

```
E   ImportError: cannot import name 'ConfusionMatrixDisplay' from 'sklearn.metrics' (/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/sklearn/metrics/__init__.py)
```